src/modules/Builder/features/ActivityItems/ActivityItems.tsx

```diff
@@ -46,6 +50,10 @@ export const ActivityItems = () => {
   );
   const itemToDelete = items[itemIndexToDelete];
   const itemName = itemToDelete?.name;
+  const conditionalLogicForItemToDelete = getItemConditionDependencies(
```

Comment:
could you pls use shorter name for variable?

Reply:
done